src/plat/bcm2711/config.cmake

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+# Copyright (C) 2021, Hensoldt Cyber GmbH
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(bcm2711 KernelPlatformRpi4 PLAT_BCM2711 KernelArchARM)
+
+if(KernelPlatformRpi4)
+    if("${KernelSel4Arch}" STREQUAL aarch32)
+        declare_seL4_arch(aarch32)
+    elseif("${KernelSel4Arch}" STREQUAL aarch64)
+        declare_seL4_arch(aarch64)
+    else()
+        fallback_declare_seL4_arch_default(aarch32)
```

Comment:
default as aarch64 since it is armv8?

Reply:
It makes sense to make AARCH64 the standard now cut some legacy. We were just unsure if anybody still has some strings attached to AARCH32 from RasPi3, so we put this in there. But I'm more than happy to remove this and define that AARCH32 must be explicitly requested.

Reply:
The RPi4 port was based on the RPi3 port. In https://github.com/seL4/seL4/blob/master/src/plat/bcm2837/config.cmake, the default architecture is also aarch32 although the cortex-a53 is ARMv8.

Reply:
I think we should ask if aarch64 might be more appropriate as the default for armv8 chips. RPi3 being aarch32 by default does not mean RPi4 should do the same. I know this is a bit nitpicking, so it is up to you :-)